tools/assets_database/main.py

```diff
@@ -1,123 +1,92 @@
-    except requests.exceptions.RequestException:
-        print(f'Couldnt download v{version} global.db from github')
-        sys.exit(1)
-
-    if response.status_code != 200:
-        print(f'Couldnt download v{version} global.db from github, got {response.status_code} status code')  # noqa: E501
-        sys.exit(1)
-
-    total_bytes = 0
-    dbpath = directory / GLOBALDB_NAME
-    chunk_size = 4096
-    print(f'Downloading v{version} globaldb from the remote...')
-    with open(dbpath, 'wb') as f:
-        for chunk in response.iter_content(chunk_size=chunk_size):
-            if chunk:
-                total_bytes += chunk_size
-                print(f'Downloaded {total_bytes * 1025} KB...')
-                f.write(chunk)
-    print('Download complete!')
-
-    return dbpath
+from .utils import clean_folder, parse_args, prepare_globaldb
 
 
-def main() -> None:
-    args = parse_args()
-    target_directory = Path.cwd() if args.target_directory is None else args.target_directory
-    target_directory = Path(target_directory)
-    if not target_directory.is_dir():
-        print(f'Given directory {target_directory} not a valid directory')
-        sys.exit(1)
-    # The way global db works it needs to be under a directory called 'global'
-    target_global_dir = target_directory / GLOBALDIR_NAME
-    target_global_dir.mkdir(parents=True, exist_ok=True)
-    get_remote_global_db(
-        directory=target_global_dir,
-        version=args.start_db,
-        branch=args.assets_branch,
-    )
+def populate_db_with_assets():
+    """Populate the globaldb created in target_directory with the updates in the remote assets repo
+    """
     print('Applying updates...')
     msg_aggregator = MessagesAggregator()
-    # We need a user db since the spam assets get synced during user unlock and
-    # they touch both the global and the user DB at the same time
-    with TemporaryDirectory():
-        globaldb = GlobalDBHandler(data_dir=target_directory, sql_vm_instructions_cb=0)
-        assets_updater = AssetsUpdater(msg_aggregator=msg_aggregator)
-        conflicts = assets_updater.perform_update(
-            up_to_version=args.target_version,
-            conflicts=None,
+    assets_updater = AssetsUpdater(msg_aggregator=msg_aggregator)
+
+    if (conflicts := assets_updater.perform_update(
+        up_to_version=parse_args().target_version,
+        conflicts=None,
+    )) is not None:
+        assert assets_updater.perform_update(
+            up_to_version=None,
+            conflicts={
+                Asset(conflict['identifier']): 'remote'
+                for conflict in conflicts
+            },
+        ) is None
+
+
+def populate_location_mappings():
+    """Apply remote updates to query mappings for assets"""
+    msg_aggregator = MessagesAggregator()
+    with TemporaryDirectory() as tmp_dir:
+        user_db = DBHandler(  # the RotkiDataUpdater takes a user db argument but is not used for the UpdateTypes that we use.  # noqa: E501
```

Comment:
If it's not used just pass `None`, leave the comment you have and if there is typin checks here do type: ignore next to the comment you left explaining it's; not used

Reply:
so.. it is used, I just checked. Earlier it was a leftover comment from  previous script. I'm removing the comment now